docs/photoniq/event-delivery/event-delivery-filters.md

```diff
@@ -0,0 +1,165 @@
+---
+sidebar_position: 60
+title: Event Delivery Filters
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+PhotonIQ Event Delivery offers advanced filtering capabilities, allowing users to specify the exact events they want to receive. By leveraging PostgreSQL `SELECT` statement syntax, users can create filters that customize the event stream to their needs. This page outlines how to construct these queries, providing syntax guidelines, use cases, and examples.
+
+## Filters syntax
+
+Filters are defined in a JSON format, which includes fields to specify the desired event criteria. The fundamental structure of a filter includes:
+
+
+- `action`: Specifies the filter action, such as "add" to add new filters or "remove" to remove existing filters. This is only supported for WebSockets since messages cannot be sent from the client to the server in SSE. Refer to [Dynamic filter management](#dynamic-filter-management) for more details.
+
+- `initialData`: When set to `TRUE`, returns the original data after subscribing to a stream. When set to `FALSE`, it only subscribes without returning the original data. The default is `FALSE`.
+- `compress`: Indicates if messages should be sent by the server in gz-compressed form. The default is `FALSE`.
+- `once`: A boolean flag (`TRUE` or `FALSE`), indicating whether the filter should be applied just once. By defualt, it is `FALSE`. 
+- `queries`: An array of SELECT statements defining the specific events to filter.
+
+The `SELECT` syntax follows standard SQL conventions, allowing for conditions (`WHERE`), logical operators (`AND`, `OR`), and specifying particular fields or using `*` for all fields within a collection or stream.
```

Comment:
Its SQL-like format. Also for nested json attributes, it can be specified like a.b.c.

Reply:
@manoj-macrometa could you provide more clarifications to this comment, I did not get the JSON part.

Reply:
E.g if I have a doc like this - 
{ 
   name: abcd,
   email: abcd@xyz.com,
   details:
   {
           phone: 12345,
           address: street 123
    }
}
And I want to subscribe if the address gets updated for person abcd. My query will be - 
select details.address where email='abcd@xyz.com'  . Here details is a nested JSON @funke-32